ludwig/utils/upload_utils.py

```diff
@@ -256,17 +258,32 @@ def upload(
         )
 
         # Upload all artifacts in model weights folder
-        upload_path = self.api.upload_folder(
+        commit_message_weights = f"{commit_message} (weights)" if commit_message else commit_message
+        commit_description_weights = f"{commit_description} (weights)" if commit_description else commit_description
+        upload_path_weights = self.api.upload_folder(
             folder_path=os.path.join(model_path, "model", "model_weights"),
             repo_id=repo_id,
             repo_type=repo_type,
-            commit_message=commit_message,
-            commit_description=commit_description,
+            commit_message=commit_message_weights,
+            commit_description=commit_description_weights,
         )
 
-        if upload_path:
-            logger.info(f"Model uploaded to `{upload_path}` with repository name `{repo_id}`")
-            return True
+        if upload_path_weights:
+            logger.info(f"Model weights uploaded to `{upload_path_weights}` with repository name `{repo_id}`")
+            commit_message_config = f"{commit_message} (config)" if commit_message else commit_message
+            commit_description_config = f"{commit_description} (config)" if commit_description else commit_description
+            upload_path_config = self.api.upload_file(
```

Comment:
@sanjaydasgupta Here too -- would you please add the type hint (and check everywhere else you made changes and look around -- please feel free to improve the existing code in this module in terms of type hints).  Thank you.